src/css/style.css

```diff
@@ -0,0 +1,274 @@
+/* -------------------Common styles--------------------------- */
+
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+main,
+footer {
+  background-color: white;
+}
+
+body {
+  background-color: lightgrey !important;
+}
+
+/* ------------------------Header------------ */
+
+.nav1,
+.sign-up,
+.tagger,
+.option-subscribe {
+  background-color: #ff0500 !important;
+}
+
+.nav2,
+.in-the-magazine,
+.user-option {
+  background-color: white !important;
+}
+
+.menus-a2 {
+  color: grey;
+}
+
+.buttons-menus {
+  font-size: 1.6rem;
+}
+
+.title-stories {
+  color: rgb(161, 153, 153);
+}
+
+.date a {
+  color: white;
+  text-decoration: none;
+}
+
+.logo {
+  height: auto;
+  width: 132px;
+}
+
+.logo1 {
+  width: 400px;
+  height: auto;
+}
+
+.subscribe {
+  color: black;
+  background-color: yellow;
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  border-radius: 11px;
+  margin: 0 10px;
+}
+
+.buttons a {
+  padding: 5px 5px;
+}
+
+.loginbutton {
+  color: white;
+  font-weight: bold;
+}
+
+.button-grey {
+  color: lightgray;
+}
+
+/* ----------------------Main section----------------------- */
+
+.featured-story p {
+  font-size: 1.5rem;
+}
+
+.category {
+  position: absolute;
+  bottom: 0.4rem;
+  left: 0;
+  padding: 0;
+}
+
+.category span {
+  font-size: 1.4rem;
+  background-color: lightgrey;
+  padding: 0.7rem 0;
+}
+
+.category a {
+  color: #ff0500;
+  text-decoration: none;
+}
+
+.tipe {
+  font-size: 1.9rem;
+  color: #ff0500;
+  text-decoration: none;
+}
+
+.img-container img {
+  width: 100%;
+  height: auto;
+}
+
+.img-container a {
+  text-decoration: none;
+}
+
+.img-container h5 {
+  font-size: 14px;
+}
+
+.description h5 {
+  font-size: 16px;
+}
+
+.description p {
+  font-size: 14px;
+}
+
+.debate,
+.formulaire {
+  width: 100% !important;
```

Comment:
Instead of `!important` you could use bootstrap width classes: https://getbootstrap.com/docs/4.0/utilities/sizing/